teams_rules.go

```diff
@@ -77,6 +79,12 @@ type UntrustedCertSettings struct {
 	Action TeamsGatewayUntrustedCertAction `json:"action"`
 }
 
+type NotificationSettings struct {
+	Enabled    bool   `json:"enabled"`
```

Comment:
https://github.com/cloudflare/cloudflare-go/blob/master/docs/conventions.md#booleans

Reply:
fixed. Although we don't have an unset state for this variable, I have made this into a pointer.